package.json

```diff
@@ -4,8 +4,10 @@
   "main": "index.js",
   "scripts": {
     "test": "ava",
-    "lint": "eslint ./",
-    "semantic-release": "semantic-release"
+    "lint": "tslint ./",
+    "build": "(rm index.d.ts index.js lib/useragent.d.ts lib/useragent.js || echo 'deleted') && tsc --declaration -p tsconfig.json",
```

Comment:
Why you create d.ts files and add them to .gitignore?

Reply:
I thought they are not source files that created automatically so they should not be included in the git, instead they are created every time building

Reply:
You can remove declarations flag